action/const.go

```diff
@@ -38,6 +40,10 @@ var (
 	ErrNilAction          = errors.New("nil action to load proto")
 	ErrInvalidAct         = errors.New("invalid action type")
 	ErrInvalidABI         = errors.New("invalid abi binary data")
+	// ErrGasTipOverFeeCap is the error when tip cap is greater than fee cap
```

Comment:
can remove this comment since the name is clear, other `err` does not have comment either